internal/services/network/express_route_circuit_peering_resource.go

```diff
@@ -159,6 +171,12 @@ func resourceExpressRouteCircuitPeering() *pluginsdk.Resource {
 							Required: true,
 						},
 
+						"ipv6_enabled": {
```

Comment:
the block name contains ipv6
```suggestion
						"enabled": {
```